src/_cffi_src/build_openssl.py

```diff
@@ -112,3 +112,15 @@ def _extra_compile_args(platform):
     libraries=_get_openssl_libraries(sys.platform),
     extra_compile_args=_extra_compile_args(sys.platform),
 )
+
+if __name__ == "__main__":
+    from cffi import recompiler
+
+    out_dir = os.getenv("OUT_DIR")
+    module_name, source, source_extension, kwds = ffi._assigned_source
+    c_file = os.path.join(out_dir, module_name + source_extension)
+    recompiler.make_c_source(ffi, module_name, source, c_file)
+
+    extra_compile_args = _extra_compile_args(sys.platform)
+    for arg in extra_compile_args:
+        print(f"cargo:rustc-link-arg={arg}")
```

Comment:
Is it OK to move `-Wconversion` and `-Wno-error=sign-conversion` to Rust side using only [flag_if_supported](https://docs.rs/cc/latest/cc/struct.Build.html#method.flag_if_supported)?

Reply:
Seems fine to me.